wbooks/src/config/api.js

```diff
@@ -2,6 +2,6 @@ import axios from 'axios';
 
 const BOOKS_URL = 'https://wbooks-api-stage.herokuapp.com/api/v1/';
 
-export const post =  (url, payload) => {
-  return axios.post(BOOKS_URL + url, payload);
-}
+export const post =  (url, payload) => axios.post(BOOKS_URL + url, payload);
```

Comment:
use template literals instead of `+`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
set the accessToken here (if it exists)